modules/auxiliary/scanner/http/exchange_web_server_pushsubscription.rb

```diff
@@ -0,0 +1,130 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'rex/proto/ntlm/message'
+require 'rex/proto/http'
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize
+    super(
+      'Name'           => 'Microsoft Exchange Privilege Escalation Exploit',
+      'Description'    => %q{
+        This module exploits a privilege escalation vulnerability found in Microsoft Exchange - CVE-2019-0724
+        Execution of the module will force Exchange to authenticate to an arbitrary URL over HTTP via the Exchange PushSubscription feature.
+        This allows us to relay the NTLM authentication to a Domain Controller and authenticate with the privileges that Exchange is configured.
+        The module is based on the work by @_dirkjan,
```

Comment:
Traditionally, authors are added to the `Author` array, with comments for attribution. For example, something like this:

```ruby
      'Author'         => [
        '@_dirkjan',         # Discovery and PoC
        'Petros Koutroumpis' # Metasploit
      ]
```

Reply:
Thanks for pointing this out. Should be okay now.